includes/migrate.php

```diff
@@ -67,7 +80,7 @@ function wp_config_requires_updating() {
 		return false;
 	}
 
-	return api_key_needs_migration( WP101_API_KEY );
+	return ! WP101_API_KEY || api_key_needs_migration( WP101_API_KEY );
```

Comment:
Because this is a constant, you could see an issue with this when it tries to evaluate `!` - this can throw an error when trying to evaluate null. Recommend changing this to the safer version:

```php
return defined('WP101_API_KEY') && ! WP101_API_KEY || api_key_needs_migration( WP101_API_KEY );
```

Depending on the expected value of `WP101_API_KEY`, we could use `! empty( WP101_API_KEY )` for additional safety.

Reply:
Good catch, though the conditionally literally right above this ensures we're returning early if the constant isn't defined ;)

Reply:
Well then....

I'm going to blame this on github's failure to show me at least a couple lines above for context.